README.md

```diff
@@ -165,7 +165,7 @@ documentation below for more information.
 The ouput file MUST use the PNG extension.
 
 ```php
-$waveform = $audio->waveform(640, 120);
+$waveform = $audio->waveform(640, 120, ['#00FF00']);
```

Comment:
Could you describe the third argument, please?

Reply:
Done

Reply:
Did you pushed it? I don't see it unfortunately. :(

Reply:
There was a bit of misunderstanding here. I though you mean to update the docblock section of the method. I have updated the README.md now. :)

Reply:
This also needs to use the `array()` syntax. :(